starlette/middleware/base.py

```diff
@@ -27,7 +27,7 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
 
     async def call_next(self, request: Request) -> Response:
         loop = asyncio.get_event_loop()
-        queue = asyncio.Queue()  # type: asyncio.Queue
+        queue = asyncio.Queue(maxsize=1)  # type: asyncio.Queue
```

Comment:
Pretty sure we don't need the type annotation comment (not sure why it's there) 🤔

Reply:
I think it's legit, in my experience otherwise we'd get a `queue needs a type annotation` error from mypy.

Reply:
It's true: mypy complains about this one without it.

Reply:
Ah I see python/mypy#4052. So we could do something like:
```suggestion
        queue: 'asyncio.Queue[Optional[Message]]' = asyncio.Queue(maxsize=1)
```

Reply:
That's fine with me. What do you think, @florimondmanca?

Reply:
Yup, looks like a nice refinement to me. Don't mind getting this in now!

Reply:
Sounds good. Thanks for the suggestion, @JayH5. I'll get that in here soon.